configs/vue3.js

```diff
@@ -1,19 +1,44 @@
 module.exports = {
+  env: {
+    browser: true,
+    node: true,
+  },
   extends: [
-    'plugin:@typescript-eslint/recommended',
+    // Standard JS rules
+    'eslint:recommended',
+    'prettier',
+    // Vue3 rules
     'plugin:vue/vue3-recommended',
-    require.resolve('./_vue-generic'),
+    'plugin:vuejs-accessibility/recommended',
+    // Typescript rules
+    'plugin:@typescript-eslint/recommended',
   ],
+  parser: 'vue-eslint-parser',
+  
+  // TS parser settings
   parserOptions: {
     parser: '@typescript-eslint/parser',
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    // Needed so TS doesn't complain about usage of browser APIs and newer JS features
+    lib: ['DOM', 'ESNext'],
   },
-  plugins: ['@typescript-eslint'],
   rules: {
-    'vue/component-tags-order': [
+    // Our base custom rules
+    'no-undefined': 'error',
+    'no-console': 'error',
+
+    // Our Vue custom rules
+    'vue/html-self-closing': [
       'error',
       {
-        order: ['script', 'template', 'style'],
```

Comment:
was wondering why this was removed, but it seems it's (pretty much) the default anyway